linux/keyman-system-service/src/KeymanSystemService.cpp

```diff
@@ -76,11 +76,22 @@ on_call_ordered_output_sentinel(
   return sd_bus_reply_method_return(msg, "");
 }
 
+static int32_t
+on_ping(
+  sd_bus_message *msg,
+  void *user_data,
+  sd_bus_error *ret_error
+) {
+  *ret_error = SD_BUS_ERROR_NULL;
+  return sd_bus_reply_method_return(msg, "s", "pong");
```

Comment:
Where is this handled?
> Verifies that the keyman-system-service is started and available

Is this not needed?
```cpp
KeymanSystemService *service = static_cast<KeymanSystemService *>(user_data);
  service->Ping();
```

Reply:
We could add a `KeymanSystemService::Ping` method, but since the method does nothing (it's only purpose is to get keyman-system-service started if isn't running already) I don't see much value in doing that.